sysl2/sysl/sysl.go

```diff
@@ -32,6 +32,8 @@ func main3(stdout, stderr io.Writer, args []string) error {
 	switch filepath.Base(args[0]) {
 	case "syslgen":
 		return DoGenerateCode(stdout, stderr, flags, args)
+	case "syslval":
```

Comment:
I'm open for suggestions 😃

Reply:
i dont want to add more of these. just add syslgen --validate.